src/selfsign.ml

```diff
@@ -25,11 +25,9 @@ let info =
   let doc = "generate a self-signed certificate" in
   let man = [ `S "BUGS"; 
               `P "Submit bugs at https://github.com/yomimono/ocaml-certify";] in
-  Term.info "selfsign" ~doc ~man
+  Term.info "selfsign" ~version:"0.0.1" ~doc ~man
```

Comment:
this should probably be extracted from the `_oasis` file, see for instance how it is done [there](https://github.com/samoht/alcotest/blob/master/Makefile#L41) (there are maybe better ways to do this)